### ISSUE TITLE
Bucket Dupe

### ISSUE THREAD

**Issue by RopeyPlague** [wyldmods/WyldCraftIssues#19]
Whenever you put water in a Crucible from Thaumcraft you get a empty bucket and keep the one you used.

**Comment by kwifrog** [comment]
Yep. this is an issue with Thaumcraft currently. Not much we can do about it :|

**Comment by ConditionOfMan** [comment]
Double confirmed. Filling Thaumic Crucible with glass water bottles dupes the empty bottles too.

**Comment by kwifrog** [comment]
As a tip, placing strips around the top of a crucible seemed to stop the bucket issues for me. It no longer dupes, a block isn't placed on top, and the bucket empties.

**Comment by tecnobrat** [comment]
This has been fixed.

**Comment by tecnobrat** [comment]
Or someone lied to me, not fixed ... either way, we can't fix this, its a bug with thaumcraft.
